tools/mesmer/mesmer.xml

```diff
@@ -43,15 +43,38 @@
                 <param name="membrane_image" type="data" format="tiff, ome.tiff" label="The path to an image containing the membrane marker(s)"/>
                 <param name="membrane_channel" type="integer" value="0"  label="The numerical index of the channel(s) from membrane-image"/>
             </when>
+            <when value="False" />
         </conditional>
     </inputs>
     <outputs>
         <data format="tiff" name="mask" from_work_dir="mask.tif" label="${tool.name} on ${on_string}: Mask"/>
     </outputs>
     <tests>
+        <test>
+            <param name="nuclear_image" value="test.tiff" />
+            <param name="compartment" value="nuclear" />
+            <param name="membrane_select.membrane_segment" value="False" />
+            <param name="image_mmp" value="0.65" />
+            <param name="squeeze" value="--squeeze" />
+            <output name="mask" ftype="tiff">
+                <assert_contents>
+                    <has_size value="350000" delta="20000" />
```

Comment:
this is a pretty big delta is that ok?

Reply:
@CameronFRWatson Can you take a look at output size for tests? When I was running, the tests were failing having a delta of 10000. Ideally we can get a closer estimate of output size.